### PR TITLE
Quote additions and deletions

### PR DESCRIPTION
**Pull request by srosebattles**
-Submitted a number of quotes, includes several for times that did not previously have a specific times.
-Deleted quotes from the Jim Carey movie "How the Grinch Stole Christmas" which didn't appear in the original Dr. Seuss book, since this is supposed to be a literature clock, not a movie clock.
-Moved and/or corrected typographical errors in several existing quotes to improve experience

### CODE REVIEW

File: litclock_annotated.csv [hunk 1]
```diff
@@ -403,10 +407,10 @@
...
+04:01|Barely past four a.m.|Charlie glanced at the clock. Barely past four a.m. at home. Sunday. She'd be up. Probably reading over a cup of coffee before taking a run before church.|Soldier On|Vanessa Rasanen|unknown
 04:01|just after 4am|Suddenly, he started to cry. Curled up on the sofa he sobbed loudly. Michel looked at his watch; it was just after 4am. On the screen a wild cat had a rabbit in its mouth.|Atomised|Michel Houellebecq|unknown
 04:02|4:02|I walked up and down the row. No one gave me a second look. Finally I sat down next to a man. He paid no attention. My watch said 4:02. Maybe he was late.|The History of Love|Nicole Krauss|unknown
 04:03|4:03 a.m.|It’s 4:03 a.m. on a supremely cold January morning and I’m just getting home. I’ve been out dancing and I’m only half drunk but utterly exhausted.|The Time Traveler’s Wife|Audrey Niffenegger|unknown
-04:03|4:03 a.m.|It's 4:03 a.m. on a supremely cold January morning and I'm just getting home. I've been out dancing and I'm only half drunk but utterly exhausted.|The Time Traveler's Wife|Audrey Niffenegger|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:409]: Removed this because it was identical in exactly every way

File: litclock_annotated.csv [hunk 2]
```diff
@@ -1069,9 +1082,11 @@
 10:53|10:53|I gaze and gaze again at that face, which seems to me both strange and familiar, said Austerlitz, I run the tape back repeatedly, looking at the time indicator in the top left-hand corner of the screen, where the figures covering part of her forehead show the minutes and seconds, from 10:53 to 10:57, while the hundredths of a second flash by so fast that you cannot read and capture them.|Austerlitz |W.G. Sebald|unknown
 10:55|10:55|“She sat down facing the empty desk, smiled at it in a way she hoped showed competence and intelligence, then sighed and slumped back and checked her watch. The interview was for eleven o’clock. It was 10:55.”|The Grammarians: A Novel|Cathleen Schine|unknown
 10:55|five minutes to eleven|The clock was still saying five minutes to eleven when Pooh and Piglet set out on their way half an hour later.|The House at Pooh Corner|AA Milne|unknown
+10:55|five minutes to eleven|Mr. Justice Wargrave said:<br>“It is now five minutes to eleven. I think we should summon Miss Brent to join our<br>conclave.”|And Then There Were None|Agatha Christie|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:1085]: This was moved from where it was, incorrectly, on line 1190

File: litclock_annotated.csv [hunk 3]
```diff
@@ -126,6 +128,7 @@
 00:57|12:57|A minute had passed, and the roller dropped a new leaf. 12:57. 12 + 57 = 69; 6 + 9 = 15; 1 + 5 = 6. 712 + 5 = 717; 71 + 7 = 78; 7 + 8 = 15; 1 + 5 = 6 again.|Lessons in Essence|Dana Standridge|unknown
 00:58|almost at one in the morning|It was downright shameless on his part to come visiting them, especially at night, almost at one in the morning, after all that had happened.|The Idiot|Fyodor Dostoyevsky|unknown
 00:59|About one o’clock|‘What time is it now?’ she said. ‘About one o’clock’. ‘In the morning?’ Herera’s friend leered at her. ‘No, there’s a total eclipse of the sun’.|Freedom|Jonathan Frantzen|unknown
+00:59|nearly one o’clock|It was nearly one o’clock when Sherlock Holmes returned from his excursion. He held in his hand a sheet of blue paper, scrawled over with notes and figures.|The Adventures of Sherlock Holmes|Sir Arthur Conan Doyle|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:131]: This quote was moved from "03:00"

**Comment by JohannesNE** [comment]
Looks very good! Thank you

**Pull request merged**